web/themes/custom/usagov/scripts/carousel.js

```diff
@@ -0,0 +1,189 @@
+jQuery(document).ready(function ($) {
+  var previousButton, nextButton;
+  var slidesContainer, slides, slideDots;
+  var leftMostSlideIndex = 0;
+
+  previousButton = document.querySelector(".previous");
+  nextButton = document.querySelector(".next");
+  slidesContainer = document.querySelector(".slides");
+  slides = slidesContainer.querySelectorAll(".slide");
+  slidesForFocus = slidesContainer.querySelectorAll(".slide a");
+  carouselHeaders = document.querySelectorAll(".carouselHeaders");
+  makeDots();
+  slideDots = document.querySelectorAll(".navigation li button");
+
+  previousButton.style.visibility = "hidden";
+  if (slideDots.length > 0) {
+    slideDots[0].setAttribute("aria-current", true);
+  }
+
+  // Set up the slide dot behaviors
+  slideDots.forEach(function (dot, index) {
+    dot.addEventListener("click", function (e) {
+      goToSlide(index);
+    });
+  });
+  // Set up previous/next button behaviors
+  previousButton.addEventListener("click", previousSlide);
+  nextButton.addEventListener("click", nextSlide);
+
+  // Ensure that all non-visible slides are impossible to reach.
+  hideNonVisibleSlides();
+
+  /** For Pagination */
+  function makeDots() {
+    var numSlides = slides.length;
+    var dots = document.getElementsByClassName("navigation")[0];
+    for (var i = 0; i < numSlides; i++) {
+      var li = document.createElement("li");
+      var pageNum = i + 1;
+      var label = `Card ${pageNum} of ${numSlides}: ${carouselHeaders[i].textContent.trim()}`;
```

Comment:
Let's also guard against (remove) quotation marks in carouselHeaders. (I am wracking my brain for other things to worry about, but that's what I've got.)